<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Deletar_campo</class>
 <widget class="QDialog" name="Deletar_campo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DELETAR CAMPO DE TABELA</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TABELA </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="NOME_TABELA">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>livros</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>livros_em_baixa</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>cadastro_pessoas</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>emprestimos</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>INDICE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="INDICE">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>11</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CHAVE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="CHAVE">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>REMOVER CAMPO</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
